reference/orders.v2.oas2.yml

```diff
@@ -1042,6 +1043,12 @@ components:
       schema:
         type: string
         default: 'application/json'
+    external_order_id:
+      name: external_order_id
+      in: query
+      description: The order ID in another system, such as the Amazon Order ID if this is an Amazon order. After setting it, you can update this field using a POST or PUT request.
+      schema: 
+        type: integer
```

Comment:
this probably should be `string` instead of `integer`

Reply:
Thanks @donald-nguyen-bc! Changing this now.